linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/bash/shared.sh

```diff
@@ -11,5 +11,22 @@ if grep -q "\(GRUB_CMDLINE_LINUX\|GRUB_CMDLINE_LINUX_DEFAULT\)" /etc/default/gru
 then
 	sed -i "s/${CONFIRM_SPAWN_YES}/${CONFIRM_SPAWN_NO}/" /etc/default/grub
 fi
+{{% if 'sle' in product %}}
+#Verify that Interactive Boot is Disabled (runtime)
+/usr/bin/grub2-editenv - unset systemd.confirm_spawn
+
+# make sure GRUB_DISABLE_RECOVERY=true
+if grep -q '^GRUB_DISABLE_RECOVERY=.*'  '/etc/default/grub' ; then
```

Comment:
Again could you move this out of the SLE specific block? I think it applies for all products.

Reply:
@vojtapolasek Absolutely, I should then update the rule to call this out.

Reply:
I can handle the needed update using grubby, but based on shared/templates/grub2_bootloader_argument/ansible.template
I don't think the code will work for ubuntu, but I don't think that the current code does either,

Reply:
I can handle the needed update using grubby, but based on shared/templates/grub2_bootloader_argument/ansible.template
I don't think the code will work for ubuntu, but I don't think that the current code does either,

Reply:
I was talking about this if expression in particular, not the Grubby approach. But you are right that Grubby probably won't work on Ubuntu.

Reply:
Updates made. Rule has:
prodtype: fedora,ol7,ol8,rhcos4,rhel7,rhel8,rhel9,rhv4,sle15

So, ubuntu is not a concern.
Also updated rule text and testing to cover GRUB_DISABLE_RECOVERY

Reply:
Updates made. Rule has:
prodtype: fedora,ol7,ol8,rhcos4,rhel7,rhel8,rhel9,rhv4,sle15

So, ubuntu is not a concern.
Also updated rule text and testing to cover GRUB_DISABLE_RECOVERY